process/datadog_checks/process/process.py

```diff
@@ -188,7 +189,7 @@ def psutil_wrapper(self, process, method, accessors, try_sudo, *args, **kwargs):
             result = {}
 
         # Ban certain method that we know fail
-        if method == 'memory_info' and Platform.is_win32() or Platform.is_solaris():
+        if method == 'memory_info' and ProcessCheck._is_win32() or Platform.is_solaris():
```

Comment:
For clarification purpose I'd name "is_32bits_windows" or something.

Reply:
Ok, will ask support to try the patch on the reproducing machine they have and then apply changes